code/modules/hallucinations/effects/blind_rush_hallucination.dm

```diff
@@ -0,0 +1,69 @@
+/**
+  * # Hallucination - Blind Rush
+  *
+  * Makes target blind and causes them to see semi-transparent humanoids running at them.
+  */
+/datum/hallucination_manager/blind_rush
+	initial_hallucination = /obj/effect/hallucination/no_delete/blind_rusher
+	trigger_time = 34 //total length of the hallucination is a little more than ten seconds
+
+
+/datum/hallucination_manager/get_spawn_location()
+	var/list/turfs = range(13, owner.loc)
+	return pick(turfs)
+
+/datum/hallucination_manager/blind_rush/on_spawn()
+	owner.playsound_local(get_turf(src), istext('sound/spookoween/ghost_whisper.ogg') ? get_sfx('sound/spookoween/ghost_whisper.ogg') : 'sound/spookoween/ghost_whisper.ogg', 30, TRUE)
+	owner.become_blind("hallucination")
+	to_chat(owner, "<span class='whisper'>Who turned off the light?</span>", MESSAGE_TYPE_INFO)
+
+/datum/hallucination_manager/on_trigger()
+	var/turf/spawn_location = get_spawn_location() //we need a new spawn location incase the player moved
+	new /obj/effect/hallucination/no_delete/blind_rusher(spawn_location, owner)
+	to_chat(owner, "<span class='danger'>They're here.</span>", MESSAGE_TYPE_INFO)
+	trigger_timer = addtimer(CALLBACK(src, PROC_REF(on_second_trigger)), trigger_time, TIMER_DELETE_ME)
+
+/datum/hallucination_manager/proc/on_second_trigger()
+	var/turf/spawn_location = get_spawn_location()
+	new /obj/effect/hallucination/no_delete/blind_rusher(spawn_location, owner)
```

Comment:
Both of these new hallucinations should be added to `hallucination_list` so they are deleted properly!